docker/README.md

```diff
@@ -0,0 +1,15 @@
+# Docker
+
+Make sure you have Docker installed
+
+## Build an image
+
+<pre>
+docker build -t knowledge -f docker/Dockerfile.dev .
```

Comment:
That command looks like you would want to run it from the root directory. With the `.` at the end you are setting the build context to root which will actually result in the build to fail at line 22 `RUN pip install -r requirements.txt`.

The reason is that in line 20 `COPY . /app/knowledge-repo-$VERSION` you are copying the given build context. in that case the whole knowledge-repo to `/app/knowledge-repo-$VERSION`. Your build will fail because requirements will not be found. You expect it to be inside the container in `/app/knowledge-repo-$VERSION/requirements.txt` but it will actually be in `/app/knowledge-repo-$VERSION/docker/requirements.txt`. The same thing would happen for `entrypoint.sh`

To fix that you need to specify proper build context and change the command to `docker build -t knowledge -f docker/Dockerfile.dev ./docker`